forms-flow-web/src/components/Form/Steps/WorkFlow.js

```diff
@@ -207,6 +207,7 @@ const WorkFlow = React.memo(
                   <Select
                     options={processList}
                     onChange={handleListChange}
+                    searchable={false}
```

Comment:
Why is searchable made false?

Reply:
by mistake added code. actually this code written for workflow search fix on select.